mivs/templates/mivs_applications/game.html

```diff
@@ -38,6 +38,9 @@
         <label class="col-sm-3 control-label">Genres:</label>
         <div class="col-sm-6">
             {% checkgroup game.genres %}
```

Comment:
Everything below here are just the text changes which were discussed over email.